README.md

```diff
@@ -44,6 +44,17 @@ new HardSourceWebpackPlugin({
     // 'debug', 'log', 'info', 'warn', or 'error'.
     level: 'debug',
   },
+  // Clean up large, old caches automatically.
+  prune: {
+    // Caches created or modified before `deleteAfter` are not considered for
+    // deletion. They must be at least this (default: 2 days) old in
+    // milliseconds.
+    deleteAfter: 2 * 24 * 60 * 60 * 1000,
```

Comment:
maybe options:

`cachePrune.maxAge`
`cachePrune.sizeThreshold`

`cachePrune` vs `prune.` is like "whatever" this is basically just a cache anyway...